src/utils/components/ComponentVersionTable.tsx

```diff
@@ -0,0 +1,33 @@
+import Text from "../../components/Text/Text";
+
+export interface ComponentVersionProps {
+  /** The DS version when the component was added*/
+  added: string;
+  /** The DS version with the most recent version of the component */
+  latest: string;
+}
+
+export const ComponentVersion = (props: ComponentVersionProps) => {
+  const { added, latest } = props;
+  const finalLatest = latest !== "Prerelease" ? `v${latest}` : latest;
+
+  return (
+    <Text
+      bg="var(--nypl-colors-ui-gray-xx-light-cool)"
```

Comment:
Why not use the design tokens?

Reply:
I prefer lighter values. But you are right...I should use our tokens.